GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -820,11 +822,11 @@ private bool DeleteSelectedFiles()
                 {
                     var path = _fullPathResolver.Resolve(item.Name);
                     bool isDir = (File.GetAttributes(path) & FileAttributes.Directory) == FileAttributes.Directory;
-                    if (isDir)
```

Comment:
Hard to understand (if this needed, see below)

                    if (isDir)
                        if (deleteFolders)
                        {
                            Directory.Delete(path, true);
                        }
                    }
                    else if (deleteFiles)
                    {
                        File.Delete(path);
                    }